crc/aarch64/crc16_t10dif_copy_pmull.S

```diff
@@ -379,13 +379,14 @@ v_br1	.req	v5
 	.size	crc16_t10dif_copy_pmull, .-crc16_t10dif_copy_pmull
 #endif
 
-ASM_DEF_RODATA
```

Comment:
Why do you want them in the .text section now?  They were in .rodata before.  You show a cross build in the comments but cross build seems to work fine when this is in rodata like other constants.

Reply:
It worked for gcc but did not work for (apple-)clang.

Reply:
```
$ make -f Makefile.unx CC=arm64-apple-darwin-gcc AR=arm64-apple-darwin-ar arch=aarch64 host_cpu=aarch64 DEFINES="-fno-stack-check" lib programs/igzip -j8
mkdir -p bin
  ---> Building crc/aarch64/crc16_t10dif_pmull.S  aarch64 
  ---> Building crc/aarch64/crc16_t10dif_copy_pmull.S  aarch64 
crc/aarch64/crc16_t10dif_pmull.S:222:9: error: unknown AArch64 fixup kind!
        ldr q_fold_const, fold_constant
        ^
crc/aarch64/crc16_t10dif_copy_pmull.S:231:9: error: unknown AArch64 fixup kind!
        ldr q_fold_const, fold_constant
        ^
make: *** [bin/crc16_t10dif_pmull.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [bin/crc16_t10dif_copy_pmull.o] Error 1
```

Reply:
this has been confirmed by https://github.com/intel/isa-l/pull/226#issuecomment-1972533399